src/main/java/org/umssdiplo/automationv01/core/managepage/home/SSIAHome.java

```diff
@@ -70,4 +77,13 @@ public WorkItemsMenu clickWorkItemsMenu() {
         CommonEvents.clickButton(workItemMenu);
         return new WorkItemsMenu();
     }
+
+    public SSIAHome(){
+        CommonEvents.isPresent(ssiaHomeContainer);
```

Comment:
Use "CommonEvents.isVisible(ssiaHomeContainer);" instead of "CommonEvents.isPresent(ssiaHomeContainer);"